app.py

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/env python3
-from flask import Flask
+from flask import Flask, render_template
 
 app = Flask(__name__)
 
 
-@app.route('/')
+''''@app.route('/')
 def hello_world():
-    return 'Hello World'
+   return 'Hello World'
+'''
+
+@app.route('/')
+def Home():
+    return render_template('Home.html')
+	
+@app.route('/')
+def Subscribe():
+    return render_template('#')
```

Comment:
Remove it. It is not part of this PR.  The instructions I provide you were wrong :-)

Reply:
I put it to see navbar with 2 categories at least. we can update the rout later when the page will be implement. what do tou think?